module/VuFind/tests/unit-tests/src/VuFindTest/Recommend/SwitchTabTest.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * SwitchTab Test Class
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+namespace VuFindTest\Recommend;
+
+use VuFind\Recommend\SwitchTab;
+
+/**
+ * SwitchTab Test Class
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+class SwitchTabTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * Data provider for testGetActiveTab.
+     *
+     * @return array
+     */
+    public function tabConfigProvider(): array
+    {
+        return ['Test1' => [[["id" => "A01",
```

Comment:
Might be worth giving the tests more meaningful names -- e.g. "Test1" could be "First tab selected" and "Test2" could be "No tab selected."